client/docker_creds_.py

```diff
@@ -149,10 +150,16 @@ def Get(self):
 
     bin_name = 'docker-credential-{name}'.format(name=self._name)
     logging.info('Invoking %r to obtain Docker credentials.', bin_name)
-    p = subprocess.Popen([bin_name, 'get'],
-                         stdout=subprocess.PIPE,
-                         stdin=subprocess.PIPE,
-                         stderr=subprocess.STDOUT)
+    try:
+      p = subprocess.Popen([bin_name, 'get'],
+                           stdout=subprocess.PIPE,
+                           stdin=subprocess.PIPE,
+                           stderr=subprocess.STDOUT)
+    except OSError as e:
+      if e.errno == errno.ENOENT:
+        raise Exception("executable not found: %s" % bin_name)
```

Comment:
nit: might as well make this a concatenation

Reply:
Done. Also made the quotes match the surrounding code. Thanks for the review!